app/Http/Controllers/ConferencesController.php

```diff
@@ -47,28 +47,34 @@ public function index(Request $request)
 
         switch ($request->input('sort')) {
             case 'date':
+                $conferences = $conferences->sortBy->starts_at;
+                break;
+            case 'opening_next':
```

Comment:
Would this sort a cfp start date in the past at the top? If so, I think we need to make the same modification here

Reply:
I modified this so when sorting by date, past conferences will now sort below future conferences.